agent/firewall.go

```diff
@@ -255,7 +258,7 @@ func (fw *Firewall) CreateU32Rules(chain int) error {
 		log.Print("Creating U32 firewall rules failed")
 		return err
 	}
-	log.Print("Creating U32 firewall rules failed for chain", chain)
+	log.Print("Creating U32 firewall rules success")
```

Comment:
u32 rules should only be applied to the FORWARD chain. INPUT and OUTPUT don't need that rule.